deluge/core/alertmanager.py

```diff
@@ -50,11 +50,10 @@ def __init__(self):
         # handlers is a dictionary of lists {"alert_type": [handler1,h2,..]}
         self.handlers = {}
         self.delayed_calls = []
-        self.wait_on_handler = False
 
     def update(self):
         self.delayed_calls = [dc for dc in self.delayed_calls if dc.active()]
-        self.handle_alerts(wait=self.wait_on_handler)
```

Comment:
Ah you are reverting a change you made a while ago, that might be what I was thinking of. Do you think we need to retain the `wait` argument for `handle_alerts()`?